starter-code/index.html

```diff
@@ -0,0 +1,253 @@
+                        paragraph that
+                        is contained to a max width
+                        by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph
+                        that is
+                        contained to a max width by a container.
+                    </div>
+                    <div class="col-sm">
+                        <img src="img/slider-2.jpg" class="img-fluid" alt="Responsive image">
+                        poppies
+
+                        A short paragraph that is contained to a max width by a container. A short paragraph that is
+                        contained to a max
+                        width by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph that
+                        is contained to a max width
+                        by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph
+                        that is
+                        contained to a max width by a container.
+                    </div>
+                    <div class="col-sm">
+                        <img src="img/slider-3.jpg" class="img-fluid" alt="Responsive image">
+                        hiking
+
+                        A short paragraph that is contained to a max width by a container. A short paragraph that is
+                        contained to a max
+                        width by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph that
+                        is contained to a max width
+                        by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph
+                        that is
+                        contained to a max width by a container.
+                    </div>
+                    <div class="col-sm">
+                        <img src="img/slider-4.jpg" class="img-fluid" alt="Responsive image">
+                        dogie
+
+                        A short paragraph that is contained to a max width by a container. A short paragraph that is
+                        contained to a max
+                        width by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph that
+                        is contained to a max width
+                        by a container. A short paragraph that is contained to a max width by a container. A short
+                        paragraph
+                        that is
+                        contained to a max width by a container.
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- bottom gray-->
+    <div id="bottom">
```

Comment:
This section still needs some styling.
Setting the width to 100% could already help with that.